src/test/kotlin/com/github/jengelman/gradle/plugins/shadow/transformers/MergeLicenseResourceTransformerTest.kt

```diff
@@ -0,0 +1,161 @@
+package com.github.jengelman.gradle.plugins.shadow.transformers
+
+import assertk.assertThat
+import assertk.assertions.containsExactlyInAnyOrder
+import assertk.assertions.isEmpty
+import assertk.assertions.isEqualTo
+import assertk.assertions.isFalse
+import assertk.assertions.isNotEqualTo
+import assertk.assertions.isTrue
+import java.io.ByteArrayOutputStream
+import java.nio.file.Path
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.io.TempDir
+
+class MergeLicenseResourceTransformerTest : BaseTransformerTest<MergeLicenseResourceTransformer>() {
+  @Test
+  fun defaultIncludes() {
+    with(transformer) {
+      assertThat(canTransformResource("META-INF/LICENSE")).isTrue()
+      assertThat(canTransformResource("META-INF/LICENSE.txt")).isTrue()
+      assertThat(canTransformResource("META-INF/LICENSE.md")).isTrue()
+      assertThat(canTransformResource("LICENSE")).isTrue()
+      assertThat(canTransformResource("LICENSE.txt")).isTrue()
+      assertThat(canTransformResource("LICENSE.md")).isTrue()
+      assertThat(canTransformResource("something else")).isFalse()
+    }
+  }
+
+  @Test
+  fun customIncludes() {
+    with(transformer) {
+      include("META-INF/FOO")
+      exclude("META-INF/LICENSE*")
+      exclude("LICENSE*")
+      assertThat(canTransformResource("META-INF/FOO")).isTrue()
+      assertThat(canTransformResource("META-INF/LICENSE")).isFalse()
+      assertThat(canTransformResource("META-INF/LICENSE.txt")).isFalse()
+      assertThat(canTransformResource("META-INF/LICENSE.md")).isFalse()
+      assertThat(canTransformResource("LICENSE")).isFalse()
+      assertThat(canTransformResource("LICENSE.txt")).isFalse()
+      assertThat(canTransformResource("LICENSE.md")).isFalse()
+      assertThat(canTransformResource("something else")).isFalse()
+    }
+  }
+
+  @Test
+  fun customIncludesWithDefaults() {
+    with(transformer) {
+      include("META-INF/FOO")
```

Comment:
Dupliates with `customIncludes`.

I guess you wanted to test a new include list here? Try out `setIncludes`. Or I believe we don't need this actually.

Reply:
Yea - let's nuke this one. It's rather a duplicate now.